core/src/test/java/com/sequenceiq/cloudbreak/controller/StackCreatorServiceTest.java

```diff
@@ -399,6 +431,158 @@ public void testFillInstanceMetadataWhenManager() {
         validateInstanceMetadataSubnetAndAvailabilityZoneAndRackId("worker", 3, hostGroupInstances.get("worker"), null, null, null);
     }
 
+    @Test
+    public void testfillInstanceMetadataTestWhenSubnetAndAZAssignmentForEnterprise() throws IOException {
```

Comment:
I think now this method name is much better but can we also mention what will be the endgoal of the test? e.g testfillInstanceMetadataTestWhenSubnetAndAZAssignmentForEnterpriseThenShouldDistributeAuxiliaryNodesAcrossAzs

Reply:
addressed.